.github/workflows/deploy-storybook.yml

```diff
@@ -0,0 +1,22 @@
+name: Build and Deploy
```

Comment:
Besides just copying this action there is some more configuration to be done to make this work. Please follow this README: https://github.com/JamesIves/github-pages-deploy-action

Reply:
Also the Hype config: https://github.com/Taraxa-project/hype/settings/pages